core/src/main/java/de/jplag/options/JPlagOptions.java

```diff
@@ -225,6 +258,46 @@ private Integer normalizeMinimumTokenMatch(Integer minimumTokenMatch) {
         return (minimumTokenMatch != null && minimumTokenMatch < 1) ? Integer.valueOf(1) : minimumTokenMatch;
     }
 
+    public interface PreParseHook {
```

Comment:
You could simply use the Functional Interface Consumer<List<Submission>>

This would reduce the amount of newly created interfaces.

Reply:
Yes makes sense, that fits the goal of these hooks exactly.